src/hooks/usePersistentAspectRatio.ts

```diff
@@ -16,7 +16,7 @@ enum State {
 
 type Result = {
   state: State;
-  result: number;
+  result: number | undefined;
```

Comment:
Is this in scope of this PR?

Reply:
Yes! Updating RN => Updating MMKV => issues with typings!